modules/administration-guide/pages/collecting-logs-using-chectl.adoc

```diff
@@ -15,7 +15,7 @@ Following commands are available to collect {prod} logs from the {orch-name} clu
 Collects existing {prod} server logs and stores them in a directory on the local machine.
 By default, logs are downloaded to a temporary directory on the machine.
 However, this can be overwritten by specifying the `-d` parameter.
-For example, to download {prod-short}:wq:q logs to the `/home/user/che-logs/` directory, use the command
```

Comment:
I love this "typo" :)

Reply:
"The editor wrote back to the content buffer strings that were intended for the command buffer"